build/Branding.props

```diff
@@ -1,6 +1,6 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <BuildName>cli</BuildName>
+    <BuildName>cli217xx</BuildName>
```

Comment:
This seems strange. Is this intentional?

Reply:
What does this do?

Reply:
@mmitche to comment, since he updated it for the previous release branches.

Reply:
From the commit for the `release/2.1.6xx` commit:

> Avoids manifest merge issues since we're now building both 2.1.5xx and 2.1.6xx in parallel in prodcon

Reply:
@livarcocc or @nguerrera does that help resolve this question?

Reply:
Yes that's correct.

Reply:
Yes, that resolved it for me. :)